courses/machine_learning/intro_to_tensorflow/2_dataset_api.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>for some reason these bullets of learning objectives didn't format correctly for me in the notebook. I think it is a spacing issue....or maybe it's just my notebook? When i changed the spacing to just one space (instead of tab) it created bullets in the markdown.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done!</p>